pkg/ebpf/processor_proctree.go

```diff
@@ -44,11 +45,13 @@ func (t *Tracee) procTreeForkProcessor(event *trace.Event) error {
 	if err != nil {
 		return err
 	}
-	forkFeed.ParentStartTime, err = parse.ArgVal[uint64](event.Args, "parent_process_start_time")
+	parentStartTime, err := parse.ArgVal[time.Time](event.Args, "parent_process_start_time")
```

Comment:
Maybe I overlooked this change in the rest of the PR, anyway, why the arg in this stage already has a time.Time instead of uint64?

Reply:
Because a motivation of the PR was enabling "decode time processing". So arguments representing some timestamps can be outputted directly as timestamps, and more importantly, normalized. This should help avoid duplicate normalization in analyze mode.
So you can see i've changed the event definition respectively.